src/chart/AccessibilityManager.ts

```diff
@@ -92,8 +92,11 @@ export class AccessibilityManager {
     const { x, y, height } = this.container.getBoundingClientRect();
     const { coordinate } = this.coordinateList[this.activeIndex];
 
-    const pageX = x + coordinate;
-    const pageY = y + this.offset.top + height / 2;
+    const scrollOffsetX = (window && window.scrollX) || 0;
```

Comment:
This would be equivalent, right? 
```suggestion
    const scrollOffsetX = window?.scrollX || 0;
```

Reply:
Amended. Thanks.